### DISCUSSION TITLE
z/OS Open Source Guild Meeting #6 - January 25, 2023 at 11am EST

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#107] [Guild]
### Agenda

* Welcome to the new year! 
  * Where we were to where we stand, to where we want to be
* Highlights in z/OS Open Source since last month (10 mins)
  * [Progress report](https://github.com/ZOSOpenTools/meta/blob/main/docs/Progress.md)
  * Introduce new tools
  * Notable zopen framework updates
* Overview of Bash and Git on z/OS
* Feedback discussion (5 mins)
* Survey and Open Discussion (5 mins)

### How to Join?
Join link: https://ibm.webex.com/ibm/j.php?MTID=m72857a0933fbcd1c58c116a13c9f1e7b 
Webinar number: 145 105 4264
Webinar password: QamxS8QX2m7 (72697879 from phones)

### Date and Time
* Jan. 25, 11am-12pm EST (8am PST)

### Recording/Slide/Poll/Chat Log
* [Open Source Guild Jan 2023.pptx](https://github.com/ZOSOpenTools/meta/files/10502172/Open.Source.Guild.Jan.2023.pptx)
* [Poll Results](https://github.com/ZOSOpenTools/meta/files/10502208/poll-results.1.pdf)
* [Video Recording](https://ibm.box.com/s/l795gouwio3vfj9ufv02imyc1uhrpw9e)

**Comment by IgorTodorovskiIBM** [comment]
The recording, slides, and poll results are now available

**Reply by netskink** [reply]
Is there a way to join the next one?

**Comment by IgorTodorovskiIBM** [comment]
Yep, we'll post info on the next Guild meeting soon! If you prefer, send me your email and I can add your to our invite list.

**Reply by netskink** [reply]
John.davis@broadcom.com
